### PR TITLE
Authentication rules

### PR DESCRIPTION
**Pull request by kyleferguson**

### CODE REVIEW

**Comment by kyleferguson** [comment]
@jlaswell ready for review. I think we can hold off on the `organizations` concept for now. We will basically have just 2 rules that determine if a user can authenticate

1. Are they the first user; if yes, 👍 , if no then:
2. Is their Github username in the database (representing they were invited), if yes then 👍 , if not then no go

File: auth/user.go [hunk 1]
```diff
@@ -7,8 +7,10 @@ import (
 // User represents a Brizo user
 type User struct {
 	database.Model
-	Username  string `gorm:"not null;unique_index"`
-	FirstName string
-	LastName  string
-	Email     string
+	Username       string `gorm:"not null;unique_index"`
```
Comment (jlaswell) [inline auth/user.go:10]: Do you think there is value in adding an id column w/ a primary key to this?

**Review by jlaswell — CHANGES_REQUESTED** [review]
Single question, but implementation looks 👍

File: auth/user.go (hunk 1 above)
Comment (kyleferguson) [reply auth/user.go:10]: @jlaswell auto ID will be applied as it is inherited from `database.Model` (see database/model.go)

**Review by jlaswell — APPROVED** [review]

**Pull request merged**